src/store/modules/ProfileModule.ts

```diff
@@ -120,7 +119,7 @@ export default {
     },
 
     mutations: {
-        resetLocalState(state: State) {
+        reset(state: State) {
             state.modList = [];
```

Comment:
I just now noticed that this doesn't reset the `activeProfile`. Why's that?

Reply:
this was the thing that got broken (trying to load the settings on first load and so on). I think we want to keep the selected profile selected if the user returns to game selection and selects the same game that was selected previously.
IDK... It seems to break more stuff to try and reset it, than just not resetting it. I think the're no cases where persisting the selected profile would cause any issues. If you feel this is something to try and dig deeper and try to fix LMK, maybe even creating a separate task for that would be smart.